src/le.py

```diff
@@ -2035,6 +2069,8 @@ def __init__(self):
         # Enabled fine-grained logging
         self.debug = False
         # All recognized events are logged
+        self.debug_cmds = False
```

Comment:
debug_cmds seems like a really generic name. What does this setting do? Can we pick a better name to reflect its behaviour?